templates/github/.github/workflows/ci.yml.j2

```diff
@@ -43,6 +43,11 @@ jobs:
       {%- endif %}
   {%- endif %}
 
+  {%- if is_pulpdocs_member %}
+  docs:
+    uses: "./.github/workflows/docs.yml"
+  {%- endif %}
+
```

Comment:
```suggestion
  {%- if is_pulpdocs_member %}
  
  docs:
    uses: "./.github/workflows/docs.yml"
  {%- endif %}

```

Reply:
Or whatever it takes to get the nice extra linebreaks...
(See how the `{%-` works...)